### ISSUE TITLE
Acceptance tests on Chrome on Browserstack not working

### ISSUE THREAD

**Issue by tatsta** [Codeception/Codeception#5546]
Not sure if this is a Codeception issue or Browserstack issue.
This is related to the new release of facebook/php-webdriver 
https://github.com/facebook/php-webdriver/blob/community/CHANGELOG.md
where they `Force Chrome to not use W3C WebDriver protocol.`
#### What are you trying to achieve?
I'm trying to run acceptance tests with Browserstack on Chrome browser. 

#### What do you get instead?

Since the latest release of facebook/php-webdriver (1.7.0) I get an error
``` bash
Test  tests/acceptance/purchaseCheckHappyPath3ds.feature:try_voidPurchaseCheck
                                                                                                                                                                               
  [Facebook\WebDriver\Exception\UnknownServerException] The property '#/' contains additional properties ["w3c"] outside of the schema when none are allowed in chromeOptions  
                                                                                                                                                                               
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/facebook/webdriver/lib/Exception/WebDriverException.php:114
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/facebook/webdriver/lib/Remote/HttpCommandExecutor.php:331
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/facebook/webdriver/lib/Remote/RemoteWebDriver.php:140
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Module/WebDriver.php:1463
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Module/WebDriver.php:426
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Subscriber/Module.php:56
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/event-dispatcher/EventDispatcher.php:260
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/event-dispatcher/EventDispatcher.php:235
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/event-dispatcher/EventDispatcher.php:75
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/phpunit-wrapper/src/Listener.php:133
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/phpunit-wrapper/src/Listener.php:102
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/phpunit/phpunit/src/Framework/TestResult.php:386
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Test/Test.php:75
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/phpunit/phpunit/src/Framework/TestSuite.php:733
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/phpunit-wrapper/src/Runner.php:110
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/SuiteManager.php:158
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Codecept.php:192
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Codecept.php:159
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Command/Run.php:383
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/console/Command/Command.php:255
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/console/Application.php:969
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/console/Application.php:255
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/symfony/console/Application.php:148
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/src/Codeception/Application.php:107
/Users/tatjana.starcenko/Projects/paymentSDK-php/vendor/codeception/codeception/codecept:43
```
Everything works fine when running locally or on Firefox

### Details

* Codeception version: 3.0.1
* PHP Version: 7.1.23
* Operating System: macOS Mojave 10.14.5
* Installation type: Composer
* List of installed packages (`composer show`)

behat/gherkin                      v4.6.0     Gherkin DSL parser for PHP 5.3
clue/stream-filter                 v1.4.1     A simple and modern approach to stream filtering in PHP
codacy/coverage                    1.4.2      Sends PHP test coverage information to Codacy.
codeception/codeception            3.0.1      BDD-style testing framework
codeception/phpunit-wrapper        6.0.16     PHPUnit classes used by Codeception
codeception/stub                   2.1.0      Flexible Stub wrapper for PHPUnit's Mock Builder
doctrine/instantiator              1.2.0      A small, lightweight utility to instantiate objects in PHP without invoking their constructors
facebook/webdriver                 1.7.0      A PHP client for Selenium WebDriver
gitonomy/gitlib                    v1.0.4     Library for accessing git
guzzlehttp/guzzle                  6.3.3      Guzzle is a PHP HTTP client library
guzzlehttp/promises                v1.3.1     Guzzle promises library
guzzlehttp/psr7                    1.5.2      PSR-7 message implementation that also provides common utility methods
hamcrest/hamcrest-php              v2.0.0     This is the PHP port of Hamcrest Matchers
hoa/consistency                    1.17.05.02 The Hoa\Consistency library.
hoa/console                        3.17.05.02 The Hoa\Console library.
hoa/event                          1.17.01.13 The Hoa\Event library.
hoa/exception                      1.17.01.16 The Hoa\Exception library.
hoa/file                           1.17.07.11 The Hoa\File library.
hoa/iterator                       2.17.01.10 The Hoa\Iterator library.
hoa/protocol                       1.17.01.14 The Hoa\Protocol library.
hoa/stream                         1.17.02.21 The Hoa\Stream library.
hoa/ustring                        4.17.01.16 The Hoa\Ustring library.
mikey179/vfsstream                 v1.6.6     Virtual file system to mock the real file system in unit tests.
mockery/mockery                    1.2.2      Mockery is a simple yet flexible PHP mock object framework
monolog/monolog                    1.24.0     Sends your logs to files, sockets, inboxes, databases and various web services
myclabs/deep-copy                  1.9.1      Create deep copies (clones) of your objects
myclabs/php-enum                   1.7.1      PHP Enum implementation
php-http/client-common             2.0.0      Common HTTP Client implementations and tools for HTTPlug
php-http/discovery                 1.6.1      Finds installed HTTPlug implementations and PSR-7 message factories
php-http/guzzle6-adapter           v2.0.1     Guzzle 6 HTTP Adapter
php-http/httplug                   v2.0.0     HTTPlug, the HTTP client abstraction for PHP
php-http/message                   1.7.2      HTTP Message related tools
php-http/message-factory           v1.0.2     Factory interfaces for PSR-7 HTTP Message
php-http/promise                   v1.0.0     Promise used for asynchronous HTTP requests
phpdocumentor/reflection-common    1.0.1      Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock  4.3.1      With this component, a library can provide support for annotations via DocBlocks or otherwise retrieve information that is embedded in a DocBlock.
phpdocumentor/type-resolver        0.4.0     
phpspec/prophecy                   1.8.0      Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage          4.0.8      Library that provides collection, processing, and rendering functionality for PHP code coverage information.
phpunit/php-file-iterator          1.4.5      FilterIterator implementation that filters files based on a list of suffixes.
phpunit/php-text-template          1.2.1      Simple template engine.
phpunit/php-timer                  1.0.9      Utility class for timing
phpunit/php-token-stream           2.0.2      Wrapper around PHP's tokenizer extension.
phpunit/phpunit                    5.7.27     The PHP Unit Testing framework.
phpunit/phpunit-mock-objects       3.4.4      Mock Object library for PHPUnit
psr/cache                          1.0.1      Common interface for caching libraries
psr/http-client                    1.0.0      Common interface for HTTP clients
psr/http-message                   1.0.1      Common interface for HTTP messages
psr/log                            1.1.0      Common interface for logging libraries
ralouphie/getallheaders            2.0.5      A polyfill for getallheaders.
robrichards/xmlseclibs             3.0.3      A PHP library for XML Security
sebastian/code-unit-reverse-lookup 1.0.1      Looks up which function or method a line of code belongs to
sebastian/comparator               1.2.4      Provides the functionality to compare PHP values for equality
sebastian/diff                     1.4.3      Diff implementation
sebastian/environment              2.0.0      Provides functionality to handle HHVM/PHP environments
sebastian/exporter                 2.0.0      Provides the functionality to export PHP variables for visualization
sebastian/global-state             1.1.1      Snapshotting of global state
sebastian/object-enumerator        2.0.1      Traverses array structures and object graphs to enumerate all referenced objects
sebastian/recursion-context        2.0.0      Provides functionality to recursively process PHP variables
sebastian/resource-operations      1.0.0      Provides a list of PHP built-in functions that operate on resources
sebastian/version                  2.0.1      Library that helps with managing the version number of Git-hosted PHP projects
squizlabs/php_codesniffer          3.4.2      PHP_CodeSniffer tokenizes PHP, JavaScript and CSS files and detects violations of a defined set of coding standards.
symfony/browser-kit                v4.3.1     Symfony BrowserKit Component
symfony/console                    v3.4.28    Symfony Console Component
symfony/css-selector               v4.3.1     Symfony CssSelector Component
symfony/debug                      v3.4.28    Symfony Debug Component
symfony/dom-crawler                v4.3.1     Symfony DomCrawler Component
symfony/event-dispatcher           v4.3.1     Symfony EventDispatcher Component
symfony/event-dispatcher-contracts v1.1.1     Generic abstractions related to dispatching event
symfony/finder                     v4.3.1     Symfony Finder Component
symfony/options-resolver           v4.3.1     Symfony OptionsResolver Component
symfony/polyfill-ctype             v1.11.0    Symfony polyfill for ctype functions
symfony/polyfill-mbstring          v1.11.0    Symfony polyfill for the Mbstring extension
symfony/process                    v3.4.28    Symfony Process Component
symfony/yaml                       v3.4.28    Symfony Yaml Component
webmozart/assert                   1.4.0      Assertions to validate method input/output with nice error messages.
whichbrowser/parser                v2.0.37    Useragent sniffing library for PHP
wirecard/iso-paypal-converter      1.0.0      Converts ISO 3166-2 codes to the codes the PayPal API requires

* Suite configuration:
```yml
actor: AcceptanceTester
modules:
  enabled:
    - WebDriver:
        host: 'hub-cloud.browserstack.com'
        port: 80
        url: 'http://foo.bar'
        browser: chrome 
        capabilities:
          'browserstack.user': 'foo'
          'browserstack.key': 'bar'
    - \Helper\Acceptance
```

**Comment by OndraM** [comment]
I believe this is bug on BrowserStack side. 

There is no reason why they should validate content of [vendor specific](https://w3c.github.io/webdriver/#dfn-extension-capability) capabilities. Please contact their support.

In the meantime, you can use php-webdriver <1.7.0.

**Comment by OndraM** [comment]
Temporarily fixed in php-webdriver 1.7.1 (see https://github.com/facebook/php-webdriver/issues/644 ). Let's hope BrowserStack adjust their schema validation soon.
